vaults/azurekeyvault.go

```diff
@@ -81,8 +81,18 @@ func NewAKVSecretConfig(vaultURL, msiClientID, vaultAADResourceURL string) (*AKV
 		secretVersion = urlSegments[3]
 	}
 
+	vaultHostWithScheme := fmt.Sprintf("%s://%s", parsedURL.Scheme, parsedURL.Host)
```

Comment:
Why do you need to construct the host with scheme? you can directly split the Host right?

Reply:
Thats because we pass vaultHostWithScheme to VaultURL below this line